### PR TITLE
Phanpy

### PR DESCRIPTION
**Pull request by JensTimmerman**

### CODE REVIEW

File: lib/vsc/manage/managecommands.py [hunk 1]
```diff
@@ -47,6 +47,8 @@
     warnings.simplefilter("ignore")
     import paramiko
 
+OPEN = 'lanplus'
```
Comment (stdweird) [inline lib/vsc/manage/managecommands.py]: use `LANPLUS`?

**Comment by JensTimmerman** [comment]
depends on https://github.ugent.be/hpcugent/quattor/pull/229

File: lib/vsc/manage/nodes.py [hunk 2]
```diff
@@ -325,6 +320,8 @@ def _getQuattorElementFromXML(self, xpath, path):
         of a given xpath
         """
         # parse xml
+        # optional if using xml for quattor, make sure this is on your system
+        import libxml2
```
Comment (stdweird) [inline lib/vsc/manage/nodes.py]: you can throw away the xml stuff

File: lib/vsc/manage/nodes.py [hunk 3]
```diff
@@ -688,7 +687,8 @@ def setoffline(self):
         """
         nodeofflinelist = []
         for node in self.getNodes():
-            nodeofflinelist.append(get_config('NODENAME_TPL') % {'nodeid': node.nodeid, 'clustername': node.clustername})
+            nodeofflinelist.append(get_config('NODENAME_TPL') % {'nodeid': node.nodeid, 'clustername':
+                                   node.clustername})
```
Comment (stdweird) [inline lib/vsc/manage/nodes.py]: split on the key?

**Comment by stdweird** [comment]
@JensTimmerman minor remarks, looks ok

**Pull request merged**